src/components/inputs/TextField.js

```diff
@@ -217,21 +217,21 @@ export default class TextField extends BaseInput {
   }
 
   getAccessibilityInfo() {
-    const {floatingPlaceholder, placeholder, expandable, value} = this.getThemeProps();
+    const {floatingPlaceholder, placeholder, expandable} = this.getThemeProps();
+    const accessibilityState = this.isDisabled() ? {disabled: true} : undefined;
+    let accessibilityLabel = floatingPlaceholder ? this.props.accessibilityLabel || placeholder : '';
 
-    let accessibilityLabel = floatingPlaceholder ? placeholder : undefined;
     if (this.isRequiredField()) {
-      accessibilityLabel = `${accessibilityLabel || ''}. Mandatory`;
```

Comment:
I think we did it on purpose ``${accessibilityLabel || ''}`, otherwise the reader will read `undefined` 
Why are u fixing it?

Reply:
it's added at the end of the ternary in line 222